docs/getting-started/server/events.md

```diff
@@ -273,37 +278,65 @@ pwsh setup_secrets.ps1 -clear
 
-### Configuring the webhook (optional)
+### Integrations and IntegrationConfigurations
 
-1. Edit the `servicebusemulator_config.json` file to add a subscription to the main `event-logging`
-   topic:
+Organizations can configure Integrations and IntegrationConfigurations to send events to different
+endpoints. Each handler maps to a specific integration and checks for the configuration when it
+receives an event. Currently, there are integrations/handlers for Slack and webhooks (as mentioned
+above).
 
-```json
-{
-  "Name": "events-webhook-subscription"
-}
-```
+**OrganizationIntegration**
 
-2. Restart the server-bus container to pick up these changes
+- This is the top level object that enables a specific integration for the Organization
+- It includes any properties that apply to the entire integration across all events
 
-3. Add the webhook subscription name and URL configuration to `secrets.json`
+  - For slack, it consists of the token:
 
-```json
-  "eventLogging": {
-    "azureServiceBus": {
-    "connectionString": "\"Endpoint=sb://localhost;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;\"",
-    "topicName": "event-logging",
-    "eventRepositorySubscriptionName": "events-write-subscription",
-    "webhookSubscriptionName": "events-webhook-subscription"
-    },
-    "webhookUrl": "<Optional URL here>"
-  },
-```
+    ```json
+    { "token": "xoxb-token-from-slack" }
+    ```
 
-4. Publish the new secrets to the apps:
+  - For webhooks, it is `null`. However, even though there is no configuration, an organization must
+    have a webhook `OrganizationIntegration` to enable configuration via
+    `OrganizationIntegrationConfiguration`
 
-   ```bash
-   pwsh setup_secrets.ps1 -clear
-   ```
+**OrganizationIntegrationConfiguration**
+
+- This contains the configurations specific to each `EventType` for the integration.
+- `Configuration` contains the event-specific configuration
+
+  - For Slack, this would contain what channel to send the message to:
+
+    ```json
+    { "channelId": "C123456" }
+    ```
+
+  - For Webhook, this is the URL the request should be sent to:
+
+    ```json
+    { "url": "https://api.example.com" }
+    ```
 
-5. Restart all services, including `EventsProcessor`
+- `Template` contains a template string the is expected to be filled in with the contents of the
+  actual event.
+  - The tokens in the string are wrapped in `#` characters. For instance, the UserId would be
+    `#UserId#`
+  - The `IntegrationTemplateProcessor` does the actual work of replacing these tokens with
+    introspected values from the provided `EventMessage`.
+  - Note that the template does not enforce any structure - it could be a freeform text message to
+    send via Slack, or a JSON body to send via webhook. We simply store and use it as a string for
+    the most flexibility.
+
+**OrganizationIntegrationConfigurationDetails**
```

Comment:
```suggestion
**`OrganizationIntegrationConfigurationDetails`**
```

Reply:
I think you can drop this whole section actually -- we don't really document these and I think its purpose is clear.